messages/bs-BA.json

```diff
@@ -0,0 +1,48 @@
+{
+  "store/social-networks": "crwdns133770:0crwdne133770:0",
```

Comment:
@camilavcoutinho i don't have enough context about it, but it will appear `crwdns133770:0crwdne133770:0` on the UI? Do you have a link for this page?

Reply:
Hey @igorbrasileiro ! It will only appear when we enable the pseudo language with a script, so don't worry, it will not appear on the UI. The workspace is this one: https://acholi--storecomponents.myvtex.com/admin, but you probably won't have access, because it is linked with Crowdin platform.